### PR TITLE
[ADD] connector_acp_ftp: add FTP backend support

### PR DESCRIPTION
**Pull request by dreispt**
Supersedes part of #61 
Requires #80

### CODE REVIEW

File: connector_acp_ftp/models/backend_acp.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import os
+import tempfile
+import shutil
+import ftplib
+from odoo import api, fields, models
+
+
+class BackendAcp(models.Model):
+    _inherit = "backend.acp"
+    _description = "Backend of Authorized Certification Providers"
```
Comment (max3903) [inline connector_acp_ftp/models/backend_acp.py]: @dreispt No need to set the description when inheriting.

File: connector_acp_ftp/models/backend_acp.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                        file_path = file_path[1:]
+                    file_dir, file_name = os.path.split(file_path)
+                    full_dir = os.path.join(temp_dir, file_dir)
+                    full_path = os.path.join(full_dir, file_name)
+                    os.makedirs(full_dir, exist_ok=True)
+                    with open(full_path, "w") as file_obj:
+                        file_obj.write(file_content)
+                rec._ftp_upload(temp_dir)
+                shutil.rmtree(temp_dir)
+        return super(BackendAcp, self).send(file_dict)
```
Comment (max3903) [inline connector_acp_ftp/models/backend_acp.py]:
```suggestion
            else:
                res = super(BackendAcp, self).send(file_dict)
        return res
```

File: connector_acp_ftp/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+* Open Source Integrators <https://www.@opensourceintegrators.com>
+
+    * Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
```
Comment (max3903) [inline connector_acp_ftp/readme/CONTRIBUTORS.rst:3]:
```suggestion
    * Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
    * Daniel Reis <dreis@opensourceintegrators.com>
```

File: connector_acp_ftp/README.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--chile-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-chile/tree/12.0/connector_acp_ftp
+    :alt: OCA/l10n-chile
+.. |badge4| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/236/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| 
+
+Support FTP protocol as a connector backend
```
Comment (max3903) [inline connector_acp_ftp/README.rst]:
```suggestion
This module adds support for FTP protocol to ACP connector backend.
```

File: connector_acp_ftp/readme/DESCRIPTION.rst [hunk 5]
```diff
@@ -0,0 +1 @@
+Support FTP protocol as a connector backend
```
Comment (max3903) [inline connector_acp_ftp/readme/DESCRIPTION.rst]:
```suggestion
This module adds support for FTP protocol to ACP connector backend.
```

**Review by nelsonramirezs — APPROVED** [review]

File: connector_acp_ftp/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "FTP ACP Connector",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "category": "Localization",
+    "summary": "Support FTP protocol as a connector backend.",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
```
Comment (max3903) [inline connector_acp_ftp/__manifest__.py:11]:
```suggestion
    "author": "Open Source Integrators, "
    "Serpent Consulting Services, "
    "Odoo Community Association (OCA)",
```

File: connector_acp_ftp/models/backend_acp.py (hunk 2 above)
Comment (dreispt) [reply connector_acp_ftp/models/backend_acp.py]: Changed.

File: connector_acp_ftp/readme/CONTRIBUTORS.rst (hunk 3 above)
Comment (dreispt) [reply connector_acp_ftp/readme/CONTRIBUTORS.rst:3]: Changed on CONTRIBUTORS.rst

File: connector_acp_ftp/__manifest__.py (hunk 6 above)
Comment (dreispt) [reply connector_acp_ftp/__manifest__.py:11]: Black decided this way, so I prefer to stick to black's output.

Comment (max3903) [reply connector_acp_ftp/__manifest__.py:11]: @dreispt :+1: but can you add Serpent?

**Pull request merged**